manim/mobject/geometry.py

```diff
@@ -809,3 +782,51 @@ class RoundedRectangle(Rectangle):
     def __init__(self, **kwargs):
         Rectangle.__init__(self, **kwargs)
         self.round_corners(self.corner_radius)
+
+
+class ArrowTip(VMobject):
+    CONFIG = {
+        "fill_opacity": 0,
+        "stroke_width": 0,
+        "length": DEFAULT_ARROW_TIP_LENGTH,
+        "start_angle": PI,
+    }
+
+    def get_base(self):
+        return self.point_from_proportion(0.5)
+
+    def get_tip_point(self):
+        return self.points[0]
+
+    def get_vector(self):
+        return self.get_tip_point() - self.get_base()
+
+    def get_angle(self):
+        return angle_of_vector(self.get_vector())
+
+    def get_length(self):
+        return get_norm(self.get_vector())
```

Comment:
Suggestion: I'd rather all of these be properties

Reply:
Agreed. I changed `base`, `tip_point` and `vector` in 38638475195b9940561deb1f49404e61dafba19a. Actually, `get_length` is not used anywhere -- and trying to refactor it as a `length` property would require additional work as the very same `length` is also passed via a corresponding entry in the `CONFIG` dict of `ArrowTip` (from where it is subsequently used to construct the tips...). I've removed the superfluous `get_length` for now.

Finally, changing `get_angle` to an `angle`-property reveals another problem: a `ArrowCircleTip` inherits from `Circle` which inherits from `Arc` -- and an `Arc` also has an `angle` property. I've renamed the former to `tip_angle`, see a01dd7f1f963b44651136d8a4982b85029834e4e.

Reply:
CONFIG dicts are going away soon, so we can perhaps revisit this when that happens. This is fine for now